src/gz/gz_scene.c

```diff
@@ -133,6 +133,21 @@ static int col_view_xlu_proc(struct menu_item *item,
   return 0;
 }
 
+static int col_view_wfc_proc(struct menu_item *item,
+                             enum menu_callback_reason reason,
+                             void *data)
+{
+  if (reason == MENU_CALLBACK_SWITCH_ON) {
+    settings->bits.col_view_wfc = 1;
+    gz.col_view_state = COLVIEW_RESTART;
```

Comment:
Setting `col_view_state` here will force collision view to turn on when changing this option. Checking if a restart is necessary should be done at https://github.com/Thar0/gz/blob/collision/src/gz/gz_col_view.c#L769 same as the other options that require a restart. References to `settings->bits.col_view_wfc` in `gz_col_view` should then be changed to the use the saved copy that holds the actual setting currently used, not the one requested in `settings`.